scripts/fornax1.sh

```diff
@@ -0,0 +1,75 @@
+#! /bin/bash
+
+set -x
+
+export a=
+export b=
+export c=
+
+pushd /root
+hostnamectl set-hostname node-a
+ufw disable
+swapoff -a
+apt-get -y update
+echo -e 'br_netfilter' | cat > /etc/modules-load.d/k8s.conf
+echo -e 'net.bridge.bridge-nf-call-ip6tables = 1\nnet.bridge.bridge-nf-call-iptables = 1' | cat >> /etc/sysctl.d/k8s.conf
+sysctl --system
+apt-get -y update
+apt-get install docker.io -y
+systemctl enable docker
+systemctl start docker
+apt-get -y update
+apt-get install -y apt-transport-https ca-certificates curl
+curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+echo -e 'deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main' | cat >> /etc/apt/sources.list.d/kubernetes.list
+apt-get -y update
```

Comment:
there's already "apt-get -y update" on line 21, do we need it again here?

Reply:
Based on my experience, need this line "apt-get -y update".  Because, before install new software package, need update current exist dependent package to current version.

Reply:
are both needed on line 21 and 25?

Reply:
Yes, after installing some package or software. need update again based on my experience.